internal/provider/vpc_resource.go

```diff
@@ -157,6 +170,41 @@ func (r *vpcResource) Create(ctx context.Context, req resource.CreateRequest, re
 	}
 }
 
+func (r *vpcResource) waitForVPCReadiness(ctx context.Context, id int64, timeouts timeouts.Value) (*tsClient.VPC, error) {
+	tflog.Trace(ctx, "VPCResource.waitForServiceReadiness")
+
+	defaultTimeout := 45 * time.Minute
```

Comment:
any rhyme or reason to land on 45m?

Reply:
copied this from services, VPC creation should probably be a lot lower. I'll go with 5 minutes to start maybe?

Reply:
maybe 15 or 30, 45 just seemed like an interesting timeout